tests/integration/telemetry/stats/prometheus/http/customizemetrics/customize_metrics_test.go

```diff
@@ -194,6 +214,9 @@ func buildQuery() (destinationQuery string) {
 		"source_workload":                "client-v1",
 		"source_workload_namespace":      appNsInst.Name(),
 	}
+	if protocol == "http" {
+		labels["request_operation"] = "getoperation"
```

Comment:
This test is proving that classify is not messing up grpc metrics.

We need to add something that is grpc specific. @kyessenov do we have access to grpc status so that it can be used in response classification ?

Alternatively @richardwxn you should continue to do request classification, but `request_operation` based on request.path.

Reply:
I added attribute based on grpc status

Reply:
@mandarjog @kyessenov @bianpengyuan WDYT?